pywt/_extensions/meson.build

```diff
@@ -75,12 +75,12 @@ if have_c99_complex
   c_args += ['-DHAVE_C99_COMPLEX', '-DCYTHON_CCOMPLEX=1']
 endif
 
-libc_wt = static_library('c_wt',
-  sources,
-  c_args : c_args,
-  include_directories : 'c',
-  dependencies : py_dep,
-)
+cython_args = ['-3', '--fast-fail', '--output-file', '@OUTPUT@', '--include-dir', '@BUILD_ROOT@', '@INPUT@']
+
+cython_gen = generator(cython,
+  arguments : cython_args,
+  output : '@BASENAME@.c',
+  depends : _cython_tree)
```

Comment:
I copied this pattern over from SciPy, this enables granular control over the flags passed to Cython, in this case it helped me to debug the actual error by setting `--gdb`

Reply:
Keeping this unresolved, since it is useful context.